maquetado/index.html

```diff
@@ -8,6 +8,65 @@
     <link rel="stylesheet" type="text/css" href="../application.css">
 </head>
 <body>
-    <h1 class="title">Front-End Tools</h1>
+	<h1 class="title">Hello World</h1>
+	<div class="content-tech">
+		<div class="tech">
+			<img src="../maquetado/assets/react_icon.svg" alt="" class="tech-image">
```

Comment:
You can simplify the route of the images here. You can try `assets/react_icon.svg` or `./assets/react_icon.svg`
This comment apply to every image